events/permissions.py

```diff
@@ -172,6 +181,25 @@ def has_object_permission(self, request: Request, view, obj):
         if request.method in permissions.SAFE_METHODS:
             return True
 
+        if (
+            settings.WEB_STORE_INTEGRATION_ENABLED
+            and request.method in ("PUT", "PATCH")
+            and request.data.get("web_store_merchants")
+        ):
+            data_keys = request.data.keys()
+
+            if len(data_keys) > 1:
+                return (
+                    request.user.is_superuser
+                    and utils.organization_can_be_edited_by(obj, request.user)
+                )
```

Comment:
Should this be
```
return request.user.is_superuser or utils.organization_can_be_edited_by(
                    obj, request.user
                )
```

Reply:
Changed to be similar to other financial administration permissions checks: it will be enough in this case for the user to be a superuser (and a regular admin cannot edit merchant data).